[2022-02-02 21:39:47,168] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: Question4_ZONE.local_to_gcs_task manual__2022-02-02T21:39:36.511344+00:00 [queued]>
[2022-02-02 21:39:47,190] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: Question4_ZONE.local_to_gcs_task manual__2022-02-02T21:39:36.511344+00:00 [queued]>
[2022-02-02 21:39:47,190] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-02 21:39:47,190] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-02 21:39:47,190] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-02 21:39:47,208] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2022-02-02 21:39:36.511344+00:00
[2022-02-02 21:39:47,212] {standard_task_runner.py:52} INFO - Started process 2010 to run task
[2022-02-02 21:39:47,215] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'Question4_ZONE', 'local_to_gcs_task', 'manual__2022-02-02T21:39:36.511344+00:00', '--job-id', '365', '--raw', '--subdir', 'DAGS_FOLDER/dag_questions4.py', '--cfg-path', '/tmp/tmpmpaxodci', '--error-file', '/tmp/tmpx56m_lk7']
[2022-02-02 21:39:47,216] {standard_task_runner.py:77} INFO - Job 365: Subtask local_to_gcs_task
[2022-02-02 21:39:47,287] {logging_mixin.py:109} INFO - Running <TaskInstance: Question4_ZONE.local_to_gcs_task manual__2022-02-02T21:39:36.511344+00:00 [running]> on host d574c7c5b085
[2022-02-02 21:39:47,443] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Question4_ZONE
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2022-02-02T21:39:36.511344+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-02T21:39:36.511344+00:00
[2022-02-02 21:39:47,788] {python.py:175} INFO - Done. Returned value was: None
[2022-02-02 21:39:47,822] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=Question4_ZONE, task_id=local_to_gcs_task, execution_date=20220202T213936, start_date=20220202T213947, end_date=20220202T213947
[2022-02-02 21:39:47,962] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-02 21:39:48,150] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
